examples/00-mapdl-examples/3d_notch.py

```diff
@@ -88,30 +87,21 @@
 # plate_with_hole_anum = mapdl.asba(rect_anum, circ_anum)
 cut_area = mapdl.asba(rect_anum, "ALL")  # cut all areas except the plate
 
-plotter = pv.Plotter(shape=(1, 3))
 
-plotter.subplot(0, 0)
 mapdl.lsla("S")
-mapdl.lplot(vtk=True, show_keypoint_numbering=True, plotter=plotter)
+mapdl.lplot(vtk=True, show_keypoint_numbering=True)
 mapdl.lsel("all")
 
 # plot the area using vtk/pyvista
-plotter.subplot(0, 1)
-mapdl.aplot(
-    vtk=True, show_area_numbering=True, show_lines=True, cpos="xy", plotter=plotter
-)
+mapdl.aplot(vtk=True, show_area_numbering=True, show_lines=True, cpos="xy")
```

Comment:
Yes, but it must be a `MapdlPlotter` object.

Reply:
I rather have the possibility to use a pyvista plotter too. We could detect that under the hood. What do you think?

Reply:
I can add support for that, but it will have to be passed down to the visualizer, and it might override some options that breaks visualizer features.

Reply:
Added support, output plotters are still `MapdlPlotter` though